packages/client/src/generation/generateClient.ts

```diff
@@ -411,9 +411,7 @@ export async function generateClient(options: GenerateClientOptions): Promise<vo
   }
 
   const schemaTargetPath = path.join(outputDir, 'schema.prisma')
-  if (schemaPath !== schemaTargetPath) {
-    await fs.copyFile(schemaPath, schemaTargetPath)
```

Comment:
We can no longer use `fs.copyFile`: `schemaPath` may refer to a schema folder with many `*.prisma` files to treat as a whole Prisma schema.